### DISCUSSION TITLE
Teensy 4.1 SPI Support

### DISCUSSION THREAD

**Discussion by Stevenlawrencehoriba** [zephyrproject-rtos/zephyr#62134] [General] [answered]
Can somebody confirm if SPI on teensy 4.1 port is now supported and what version of Zephyr is required for it. Or is it still being worked on? We are considering using Zephyr but we need SPI support as our custom board has various SPI chips , ADC extensions, thermocouples and DAC? is this possible now?

**Comment by henrikbrixandersen** [comment, marked as answer]
Support for the i.MX LPSPI peripheral was added in 2018, but no SPI busses are enabled by default in the Teensy 4.1 board definition. They can be enabled in a devicetree overlay, though.

**Comment by jmend3s** [comment]
Hello, can you give an example of how to enable an spi bus on the overlay file please
